haskell/BUILD.bazel

```diff
@@ -52,3 +57,8 @@ toolchain_type(
     name = "doctest-toolchain",
     visibility = ["//visibility:public"],
 )
+
```

Comment:
It should be documented in the code that this target is a workaround for https://github.com/bazelbuild/proposals/blob/master/designs/2019-02-12-toolchain-transitions.md not existing yet. Because this design is no longer going to work as-is once Bazel gets "configurations" (see [here](https://blog.bazel.build/2019/02/11/configurable-builds-part-1.html)). When that happens, there might be multiple configurations during the build, each of which require their own toolchain libraries.

Reply:
:+1: I've added a comment to that effect.